### DISCUSSION TITLE
Playbook Play Creation Issue

### DISCUSSION THREAD

**Discussion by Nolan-Mcfly** [Security-Onion-Solutions/securityonion#4195] [Unsupported Versions]
When inside SOC and the Playbook web interface, I went to test the Playbook creation tool and I was following [this](https://www.youtube.com/watch?v=IS2SOlDedPc) video about Detection Engineering. This is what happens when I try `sudo so-playbook-ruleupdate`

`sigma/rules/windows/malware/win_mal_octopus_scanner.yml
No Current Play - Create New Play in PlaybookPlay Creation Error - 
Play Creation Error - 
Play Creation Error - 
Play Creation Error -`

This problem continues on for 15+ minutes where it goes through each rule and errors out each time. 

**The steps I've taken to remedy the situation go like this:**

`so-salt-stop`
`salt-call state.apply playbook.db_init`
`so-playbook-restart`
`so-playbook-ruleupdate`

The rule update command is where things go haywire. While looking through every error that occurred, I noticed something. 

`-= Creating/Updating Plays based on the following categories: ['windows'] -=`

It seems that it is trying to create and update plays for Windows. I am not sure if this is exactly an issue or something that I need to change because it may just be adding rules that apply to Windows machines that are going to be scanned on the network for security purposes? Such as Windows Event ID 4720? 

```
sigma/rules/windows/image_load/sysmon_susp_fax_dll.yml
No Current Play - Create New Play in Playbook
{'play_creation': 401, 'play_url': 'failed'}
new


sigma/rules/windows/image_load/sysmon_susp_office_dotnet_clr_dll_load.yml
No Current Play - Create New Play in Playbook
{'play_creation': 401, 'play_url': 'failed'}
new


sigma/rules/windows/image_load/sysmon_susp_office_dotnet_assembly_dll_load.yml
No Current Play - Create New Play in Playbook
{'play_creation': 401, 'play_url': 'failed'}
new
```

You can see how each one is bringing back a 401 error, which is the same error I was receiving when attempting to create sigma rules in the playbook web interface.

Any and all comments/help are welcomed.

Thank you

**_Ubuntu 18.04
Static IP
Intel Processor
24 GB of RAM_**

--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
-= Update Summary =-

Sigma Community Repo:
 Already up to date.

Updated Plays: 0
Updates Available: 0
New Plays: 495
No Updates Needed: 0

Enabled Rulesets:
['windows']
```

**Comment by defensivedepth** [comment]
@Nolan-Mcfly Is this related to your other post I just answered? https://github.com/Security-Onion-Solutions/securityonion/discussions/4185

**Reply by Nolan-Mcfly** [reply]
Hi @defensivedepth thank you for reaching out. The issue above doesn't seem to be related to #4185, but I could be wrong. I was able to fix the previous issue with Mysql, but I am having an issue getting any rules listed in the playbook web interface, under 'all plays' it shows no data to display, and running so-playbook-ruleupdate brings back constant play creation errors.

**Reply by defensivedepth** [reply]
@Nolan-Mcfly I suspect they are related. Have you done a fresh install? I would suggest we not spend a bunch of time troubleshooting a system that has had a bunch of modifications done to it (installing mysql manually on the host system, etc)

**Reply by Nolan-Mcfly** [reply]
@defensivedepth I can do a fresh install. Just a fresh install of security onion correct? Not Ubuntu as well?

What commands are necessary to fully purge/remove all security onion and dependencies?

**Reply by defensivedepth** [reply]
@Nolan-Mcfly It sounds like things have been changed on the host os (mysql installed), so I would recommend reinstalling both Ubuntu + SO.... Or Just use the centos ISO (https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md)
